spec/factories/rules_of_origin/scheme_factory.rb

```diff
@@ -7,6 +7,7 @@
     countries               { %w[FR ES IT DE] }
     footnote                { 'This scheme may be expanded in the future' }
     unilateral              { nil }
+    cumulation_methods      { { 'bilateral' => { 'countries' => %w[GB CA] }, 'extended' => { 'countries' => %w[EU AD] }, 'diagonal' => { 'countries' => %w[EU AD] } } }
```

Comment:
This would be easier to read/modify as multiple lines rather then a long single line, eg

```ruby
cumulation_methods do
  {
    'bilateral' => %w[GB CA]
  }
end
```

The nesting under countries can be dropped because the assignment method handles both nested and unnested so this will remove one (minor) complexity from the test data